---
uint16 amplitude_threshold
